### PR TITLE
Add encryption to EBS provisioner

### PR DESCRIPTION
**Pull request by markturansky**
@jsafrane I think it is even easier this way than me making my own PR.

### CODE REVIEW

File: pkg/volume/aws_ebs/aws_util.go [hunk 1]
```diff
@@ -98,6 +98,10 @@ func (util *AWSDiskUtil) CreateVolume(c *awsElasticBlockStoreProvisioner) (strin
 			if err != nil {
 				return "", 0, nil, fmt.Errorf("invalid iopsPerGB value %q, must be integer between 1 and 30: %v", v, err)
 			}
+		case "encrypted":
+			volumeOptions.Encrypted = v
```
Comment (jsafrane) [inline pkg/volume/aws_ebs/aws_util.go:106]: It should parse "false"/"true" and throw an error when there is something else

File: examples/experimental/persistent-volume-provisioning/README.md [hunk 2]
```diff
@@ -64,6 +64,8 @@ parameters:
 * `type`: `io1`, `gp2`, `sc1`, `st1`. See AWS docs for details. Default: `gp2`.
 * `zone`: AWS zone. If not specified, a random zone in the same region as controller-manager will be chosen.
 * `iopsPerGB`: only for `io1` volumes. I/O operations per second per GiB. AWS volume plugin multiplies this with size of requested volume to compute IOPS of the volume and caps it at 20 000 IOPS (maximum supported by AWS, see AWS docs).
+* `encrypted`: denotes whether the EBS volume should be encrypted or not.
```
Comment (jsafrane) [inline examples/experimental/persistent-volume-provisioning/README.md]: please mention allowed values - is it "true"/"false" or "yes"/"no" or what?

**Comment by markturansky** [comment]
@jsafrane parsing added and additional info in readme.

I'll wait for https://github.com/kubernetes/kubernetes/pull/29006 to merge and then make a PR for these additions.

**Comment by markturansky** [comment]
closing in favor of https://github.com/kubernetes/kubernetes/pull/30880

**Pull request closed without merging**